### ISSUE TITLE
DSGVO - Brauchen wir einen Cookie-Hinweis?

### ISSUE THREAD

**Issue by NicoleRauch** [softwerkskammer/Agora#1333]
Wenn ja, was bedeutet das?

- Wir müssen so ein Cookie-Policy-Dings einblenden und entsprechend eine Seite für Detailinfos bereitstellen. Ich nehme an, da muss draufstehen, was wir für Cookies haben und wozu die gut sind und so. Und wir müssen uns pro User merken, dass diejejnige dieses Dings geklickt hat.

Können wir stattdessen was tun?

Mir ist nicht ganz klar, ob wir das wirklich brauchen. (Jemand sagte, wenn die Cookies zu rein technischen Zwecken genutzt werden, ist das nicht erforderlich.)
Ich denke, wir sollten mal eine vernünftige User-Doku in der SWK anlegen und dort einen Abschnitt "Datenschutz" machen. Dort können wir dann genau beschreiben, was für Cookies wir anlegen und wozu die verwendet werden.

**Comment by leider** [comment]
Meines Erachtens reicht es, das in der Datenschutzdoku zu erwähnen.
- Zusätzlich könnten wir die Session-Lifetime weiter reduzieren auf Kosten des Komforts

**Comment by NicoleRauch** [comment]
Naja, "auf Kosten des Komforts" triffts ja nicht so ganz, wenn man das Login-Problem anschaut und das dafür implementierte Cookie ;-)

**Comment by NicoleRauch** [comment]
Oder anders: Ich sehe keinen Sinn in einem Cookie, das "verkrüppelt" ist, aber trotzdem noch vorhanden. Das ist für mich worst of both worlds.

**Comment by leider** [comment]
Cookie brauchen wir für Anmeldung, um einen User zu identifizieren. - Also rein technische Gründe. Komfort heißt nur, nach welcher Zeit ich mich erneut anmelden muss.

**Comment by NicoleRauch** [comment]
Ja, und ich benutze ja seit relativ kurzer Zeit ein Cookie, um den zuletzt gewählten Provider im Login-Fenster anzuzeigen. Darauf bezog ich mich.
